doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -358,6 +409,11 @@ functions from ``numpy`` itself are already overloaded (but
 inadequately), so confusion about high vs. low level APIs in NumPy would
 still persist.
 
+Alternatively, a separate namespace, e.g., ``numpy.array_only``, could be
+created for a non-overloaded version of NumPy's high level API, for cases
+where performance with NumPy arrays is a critical concern. This has most
+of the same downsides as the separate namespace.
```

Comment:
I don't think the downsides are quite the same because if you use the wrappers, the separate "namespace" can be created automatically.